pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -241,3 +241,67 @@ func certificate(ns string) []unstructured.Unstructured {
 		},
 	}
 }
+
+/*
+ ***************************************
+ *** How to test a webhook locally  ****
```

Comment:
I think this belongs in `CONTRIBUTING.md` or `test/README.md` rather than here?

Reply:
Also, if this is so complex and manual, does it mean we have no integration test which runs a webhook?

Reply:
We don't. I will work on adding e2e tests in a followup PR